source/fundamentals/crud/compound-operations.txt

```diff
@@ -210,11 +228,11 @@ with the following methods:
 Example
 ```````
 
-The following example performs the following actions in order with the
+The following example performs the following actions in order using the
 ``FindOneAndUpdate()`` method:
 
-- Matches a document where the ``type`` is "Oolong"
-- Updates the matched document's ``rating`` to ``9``
+- Matches a document where the ``title`` field value includes "Modern"
```

Comment:
Issue: If this is like the Find methods in other drivers, it specifically updates the first matching document, not just an arbitrary one. Probably good to make that clear.

Suggestion:

```suggestion
- Matches the first document where the ``title`` field value includes "Modern"
```